### PR TITLE
Updated getting docker-machine properties

### PR DESCRIPTION
**Pull request by iworker**
I've got an error when running docker-machine-nfs:
```
docker-machine-nfs default --shared-folder=/Users
[INFO] Configuration:

	- Machine Name: default 
	- Shared Folder: /Users
	- Mount Options: noacl,async 
	- Force: false 

[INFO] machine presence ... 			OK 
[INFO] machine running ... 			FAIL

The machine 'default' is not running but 'virtualbox'! 
```

When I checked, the reason was updated docker-machine which returns different count of columns compared to old versions.
So, I changed behavior to get these properties without grep-sed-awk magic, using only native docker-machine command line options.

### CODE REVIEW

File: docker-machine-nfs.sh [hunk 1]
```diff
@@ -239,7 +239,7 @@ checkMachineRunning ()
 {
   echoInfo "machine running ... \t\t\t"
 
-  machine_state=$(docker-machine ls $2 | sed 1d | grep "^$1\s" | awk '{print $4}')
+  machine_state=$(docker-machine ls -t $2 --filter "Name=$1" --format "{{.State}}")
```
Comment (tonivdv) [inline docker-machine-nfs.sh]: Will this search on the exact full name? It doesn't work as a wildcard? So if for example you have a machine called "test" and a machine "test1" ... will the filter only return 1 entry when filtering for "test" ?

Comment (iworker) [reply docker-machine-nfs.sh]: I'll check it now.

Comment (iworker) [reply docker-machine-nfs.sh]: Yes, you're right. I'll add `^` and `$` to fix it.

Comment (iworker) [reply docker-machine-nfs.sh]: I fixed it.

**Comment by tonivdv** [comment]
Thanks

**Pull request merged**